qa-heal.planx-pla.net/portal/gitops.json

```diff
@@ -442,9 +440,9 @@
             }
           ]
         }
-      ],
-      "tagsDisplayName": "Study Characteristics"
+      ]
     },
+    "tagsDisplayName": "Study Characteristics",
```

Comment:
Should this also be "Study Filters"?

Reply:
no this is the display name for the `tags` column in table

it was intended to be this, but just misplaced so it wasn't in effect